draft-ietf-quic-multipath.md

```diff
@@ -1308,42 +1314,43 @@ As the MP_NEW_CONNECTION_ID frames applies the sequence number per path,
 the sequence number in the MP_RETIRE_CONNECTION_ID frame
 also needs to be considered in the context of the Path Identifier field.
 
-## MAX_PATHS frames {#max-paths-frame}
+## MAX_PATH_ID frames {#max-paths-frame}
 
-A MAX_PATHS frame (type=0x15228c0b) informs the peer of the cumulative number of paths
-it is permitted to open.
+A MAX_PATH_ID frame (type=0x15228c0c) informs the peer of the maximum path identifier
+it is permitted to use.
 
 When there are not enough unused path identifiers, endpoints SHOULD
-send MAX_PATHS frame to inform the peer that new path identifiers are available.
+send MAX_PATH_ID frame to inform the peer that new path identifiers are available.
 
-MAX_PATHS frames are formatted as shown in {{fig-max-paths-frame-format}}.
+MAX_PATH_ID frames are formatted as shown in {{fig-max-paths-frame-format}}.
 
 ~~~
-MAX_PATHS Frame {
-  Type (i) = 0x15228c0b,
-  Maximum Paths (i),
+MAX_PATH_ID Frame {
+  Type (i) = 0x15228c0c,
+  Maximum Path Identifier (i),
 }
 ~~~
-{: #fig-max-paths-frame-format title="MAX_PATHS Frame Format"}
+{: #fig-max-paths-frame-format title="MAX_PATH_ID Frame Format"}
 
-MAX_PATHS frames contain the following field:
+MAX_PATH_ID frames contain the following field:
 
 Maximum Path Identifier:
 : A count of the cumulative number of paths that can be opened
   over the lifetime of the connection. This value MUST NOT exceed 2^32-1, as
   Path IDs are defined with a maximum value 2^32-1 as the 32 bits of the Path ID are used
-  to calculate the nonce (see Section {{multipath-aead}}).
+  to calculate the nonce (see {{multipath-aead}}).
   The Maximum Paths value MUST NOT be lower than the value
-  advertised in the initial_max_paths transport parameter. Receipt
-  of an invalid Maximum Paths value MUST be treated as a
+  advertised in the initial_max_path_id transport parameter. Receipt
```

Comment:
... or what?  Is it a frame error to receive a MAX_PATH_ID with a value less than (or equal to...?) the transport parameter?